server/configs/config.js

```diff
@@ -14,8 +15,11 @@ const config = {
   },
   production: {
     url: process.env.PROD_DATABASE_URL,
-    dialect: process.env.PROD_DIALECT
+    dialect: process.env.PROD_DIALECT,
+    logging: false
   }
 };
 
+export const expiryTime = process.env.NODE_ENV === "test" ? 7 : process.env.TTL;
```

Comment:
expiration for What?
Don't hard code variables store them in constants.
What is TTL?